roles/lego/tasks/main.yml

```diff
@@ -206,6 +206,7 @@
 - name: "Enable lego.service"
   systemd:
     name: "lego.service"
+    state: "started"
```

Comment:
Please add a comment above why we do this? It's not clear at the moment why this line is here if you only look at the code.

Alternatively, use `ansible.builtin.file` to fix the permissions instead, to make it more obvious what is happening here and why.

Reply:
@jdreichmann Could you incorporate this in #179?

Reply:
I know the reason, I'm just asking for that reason to be included in the ansible code, so that people don't have to go digging when they want to understand why we start the renewal unit right after we've gotten the certificates in the first place.

With regard to handler-after-handler: Why does it need to be a handler? We're flushing the handlers right before that, so we don't need to have it in a handler. We can just ensure the permissions are right after we've flushed the handlers, which will run *after* the run lego handler in every case.

Either way, I'm fine with both approaches ultimately, just if the approach without `ansible.builtin.file` is used, then I'd want a comment explaining why this happens.